### ISSUE TITLE
About the form of the loss function

### ISSUE THREAD

**Issue by njuhang** [leftthomas/SRGAN#102]
why choose 1-D(x).mean+D(Gz)).mean() as loss function of Discriminator, rather than BCELoss? And the form of Generator loss function is different from the paper. Are the pictures and PSNR data shown here trained by this code?  Why is my training result close to the interpolation result?

**Comment by leftthomas** [comment]
@njuhang
Duplicate of #101
